README.md

```diff
@@ -18,427 +18,15 @@ A modern, lightweight learning management system.
 
 ## Getting started
 
-### Getting started on OS X
+### Getting started on MacOS, Linux or Docker.
```

Comment:
Remove period from headers. OS X is now [macOS](http://www.apple.com/macos/sierra/) -- lowercase m. 😊